axelor-production/src/main/java/com/axelor/apps/production/service/configurator/ConfiguratorProdProcessServiceImpl.java

```diff
@@ -126,14 +146,16 @@ public ProdProcess generateProdProcessService(
             stockLocation,
             producedProductStockLocation,
             workshopStockLocation,
+            isConsProOnOperation,
             product);
 
     List<ConfiguratorProdProcessLine> confLines =
         confProdProcess.getConfiguratorProdProcessLineList();
     if (confLines != null) {
       for (ConfiguratorProdProcessLine confLine : confLines) {
         ProdProcessLine generatedProdProcessLine =
-            confProdProcessLineService.generateProdProcessLine(confLine, attributes);
+            confProdProcessLineService.generateProdProcessLine(
```

Comment:
isConsProOnOperation is a Boolean (nullable), and the method `generateProdProcessLine` is using a primitive type boolean, so I think a NPE can be thrown here.